app/scss/layout/_page.scss

```diff
@@ -5,3 +5,11 @@
   min-height: 100vh;
   @include tinted-bg($black, .7, 'http://science-all.com/images/wallpapers/black-and-white-images/black-and-white-images-13.jpg', cover);
 }
+
```

Comment:
@bursteinz why create another class .pagepopup that does exactly the same thing as .page